eZ/Bundle/EzPublishSolrSearchEngineBundle/Command/SolrCreateIndexCommand.php

```diff
@@ -60,9 +60,13 @@ protected function execute( InputInterface $input, OutputInterface $output )
 
         /** @var \eZ\Publish\Core\Search\Solr\Content\Handler $contentSearchHandler */
         $contentSearchHandler = $searchHandler->contentSearchHandler();
-        $contentSearchHandler->setCommit( false );
-        $contentSearchHandler->purgeIndex();
         $contentSearchHandler->setCommit( true );
```

Comment:
nitpick, but as said before commit is by default true, so if no code is setting it to false for optimization at the moment, then this can be removed.